src/modules/bigquery.js

```diff
@@ -159,6 +160,40 @@ export default class BigQuery {
       .then((response) => response.data.rows);
   };
 
+  getAutocompleteForColumn = (
+    currText,
+    columnName,
+    dataset,
+    tableName,
+    token,
+    filterStatement,
+    joinStatement,
+  ) => {
+    const url = `https://bigquery.googleapis.com/bigquery/v2/projects/${dataset.dataProject}/queries`;
+    const filterOrEmpty = _.isEmpty(filterStatement)
+      ? `WHERE ${columnName} LIKE '%${currText}%'`
+      : `WHERE ${filterStatement} AND ${columnName} LIKE '%${currText}%'`;
+    const joinOrEmpty = _.isEmpty(joinStatement) ? '' : joinStatement;
```

Comment:
Is this just equal to `joinStatement`? I believe it is a string so in both cases `joinOrEmpty` and `joinStatement` are equal

Reply:
not quite, if there is no `joinStatement` (it hasn't been created in the store), it'll get put in as `undefined`

Reply:
Ok, got it!!